lib/matplotlib/tests/test_image.py

```diff
@@ -1252,6 +1252,23 @@ def test_huge_range_log(fig_test, fig_ref):
                    interpolation='nearest', cmap=cmap)
 
 
+@check_figures_equal(extensions=['png'])
+def test_huge_range_log(fig_test, fig_ref):
+    data = np.full((5, 5), 1, dtype=np.float64)
+    data[0:2, :] = 1E20
+
+    ax = fig_test.subplots()
+    im = ax.imshow(data, norm=colors.LogNorm(vmin=1, vmax=data.max()),
+                   interpolation='nearest', cmap='viridis')
+
+    data = np.full((5, 5), 1, dtype=np.float64)
+    data[0:2, :] = 1000
+
+    ax = fig_ref.subplots()
+    im = ax.imshow(data, norm=colors.Normalize(vmin=1, vmax=data.max()),
+                   interpolation='nearest', cmap='viridis')
```

Comment:
Pretty sure you don't need to actually duplicate the test function, just use `@pytest.mark.parametrized()`. But that is a minor nitpick, and my earlier approval still stands. This does fix an actual bug, even if it isn't the same bug everyone thought it was going to fix.

Reply:
Great suggestion, thanks!